src/commands/adivinhar.js

```diff
@@ -65,14 +65,14 @@ async function iosOrAndroidPc(interaction) {
 	return plataform;
 }
 
-const usersTries = [];
-const activeGames = [];
+const usersTries = {};
+const activeGames = {};
 
 setInterval(() => {
 	const brazilianTime = dayjs().tz('America/Sao_Paulo').format('HH:mm');
 	if (brazilianTime === '00:00') {
-		usersTries = [];
-		activeGames = [];
+		usersTries = {};
+		activeGames = {};
```

Comment:
```suggestion
		for (const obj of [userTries, activeGames]) {
			for (const key in obj) {
				delete obj[key];
			};
		};
```

Only caught this now. The current code will throw a TypeError, since both objects are constants.

Another solution to this is define them with `let`.

Reply:
Looks like you fixed it while I was reviewing. So just mark it as outdated.

Reply:
Looks like you fixed it while I was reviewing. So just mark it as outdated :)